uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -951,10 +951,12 @@ define([
 
               it('sets from time to minimum allowed time', function () {
                 expect(controller.uiOptions.times.from.time).toBe(minTime);
+                expect(leaveRequest.from_date.slice(11, 16)).toBe(minTime);
```

Comment:
I have no idea what's 11 and 16. Use this instead:

```js
expect(moment(leaveRequest.from_date).format('HH:mm')).toBe(minTime);
```

Reply:
Done.